### PR TITLE
update selection of interpolation methods

### PR DESCRIPTION
**Pull request by jimustafa**
Attempts to address #28.

- replace `'none'` with `blackman`

Something else to consider would be making the image data the same as what is used in the current matplotlib [gallery example](https://matplotlib.org/stable/gallery/images_contours_and_fields/interpolation_methods.html) for the interpolation methods. To that effect, I could add a commit to this PR.

### CODE REVIEW

**Comment by jklymak** [comment]
I actually am not sure we should advertise these so exhaustively.  It takes a lot of space for something that is pretty advanced. 

'none' otoh probably should not be dropped?

**Comment by jimustafa** [comment]
It seemed that there was some consensus in #28 that `'none'` could be dropped. For clarity, could we list the 18 interpolations that should be demonstrated.

While I agree that a 3x3 grid of 9 common interpolations may be sufficient, that does leave the question of what to put in the resulting open space. Maybe this is best addressed in a separate issue/PR (see #81).

**Comment by jklymak** [comment]
I suspect the right thing to do here is to make sure the pcolormesh objects are rasterized so that the result of "none" does not depend on the pdf viewer.  However "none" is very heavily used (for better or worse) so I think it would be odd to not document how it works in the cheat sheet.

**Comment by jklymak** [comment]
@jimustafa are we publishing the artifacts from CI anywhere?

**Comment by jimustafa** [comment]
Yes @jklymak, the cheatsheets and handouts are saved as build artifacts during workflow execution, and published to the `gh-pages` branch after being successfully built.

https://github.com/matplotlib/cheatsheets/blob/efd4431900d1ee36d6a0549b1944427002d9471d/.github/workflows/main.yaml#L57-L70

**Comment by jklymak** [comment]
but only if its a merge to master, right?

**Comment by jimustafa** [comment]
While the artifacts are saved for each build, publishing to `gh-pages` is only on merge to `master`, yes.

**Comment by jklymak** [comment]
Yeah, but I never know _where_ the artifacts are stored.  Probably just me not seeing a button somewhere...

**Comment by QuLogic** [comment]
They're on the Summary page, e.g., https://github.com/matplotlib/cheatsheets/actions/runs/1433495962

File: scripts/interpolations.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
                         subplot_kw={'xticks': [], 'yticks': []})
 for ax, interp_method in zip(axs.flat, methods):
     ax.imshow(Z, interpolation=interp_method, cmap='viridis',
-              extent=[0,9,0,9])
+              extent=[0,9,0,9], rasterized=True)
```
Comment (jimustafa) [inline scripts/interpolations.py:16]: Hey @jklymak. Is this what you were suggesting?

Comment (jklymak) [reply scripts/interpolations.py:16]: I think so.   Did it work?  After suggesting it, I wasn't sure if `imshow` rasterized, but I'm pretty sure it does.

**Comment by jklymak** [comment]
This looks fine in my viewer.  I think rasterizing it is the right way to go to make it clear what the differences are.

interp

OTOH, the reader is definitely going to be confused as to what None/none/nearest all mean in this context.  If I were doing this I would have a section:

```
# Image interpolation (over-sampled)

"nearest"/"none" = default ("antialiased", `None`), "bilinear"; "bicubic"; etc..
```
and a different section

```
# Image interpolation (under-sampled)

## `interpolation_stage="data"`:

"antialiased" = default(`None`); "none"; "hanning"; etc (the other low-pass filters don't matter a ton)

## `interpolation_stage="rgba"`:

"antialiased" = default(`None`); "none"; "hanning"; etc (the other low-pass filters don't matter a ton)

```

the fact of the matter is that for over-sampled images (many more pixels than data) "none" is a perfectly sensible choice.  For under-sampled data (fewer pixels than data) "none" is almost always aliased and should be low-passed first.

**Review by timhoffm — APPROVED** [review]
I think this is the right fix for a consistent display.

**Pull request merged**